src/søknad/steg/5-aktivitet/Aktivitet.tsx

```diff
@@ -106,53 +106,29 @@ const Aktivitet: React.FC = () => {
         />
       </SeksjonGruppe>
 
-      {erAktivitetHuketAv(ArbeidssituasjonType.erHjemmeMedBarnUnderEttÅr) && (
-        <HjemmeMedBarnUnderEttÅr />
-      )}
-
-      {erAktivitetHuketAv(ArbeidssituasjonType.etablererEgenVirksomhet) && (
-        <EtablererEgenVirksomhet
-          arbeidssituasjon={arbeidssituasjon}
-          settArbeidssituasjon={settArbeidssituasjon}
-        />
-      )}
+      {arbeidssituasjon.hvaErDinArbeidssituasjon.svarid.map((svarid) => {
+        const harValgtFlereEnnEn = hvaErDinArbeidssituasjon.svarid.length !== 0;
```

Comment:
For meg er dette "harValgtFlereEnn0" - eller?

Reply:
Glemte å oppdatere navnet etter å ha endret funksjonen, fikser ✨